packages/wallet-sdk/src/sign/scw/message/type/Action.ts

```diff
@@ -127,6 +131,36 @@ export type AddEthereumChainAction = {
   };
 };
 
+export type GetCapacitiesAction = {
+  method: SupportedEthereumMethods.WalletGetCapacities;
+  params: null;
+};
+
+export type WalletSendTransactionAction = {
+  method: SupportedEthereumMethods.WalletSendTransaction;
+  params: {
+    chainId: number;
```

Comment:
think this should be hex

Reply:
thanks, addressed in [4c4ccb7](https://github.com/coinbase/coinbase-wallet-sdk/pull/1138/commits/4c4ccb74d650bdcbbe02ef34aba545d8ce449104)